CONTRIBUTING.md

```diff
@@ -2,6 +2,10 @@
 
 ## Getting Started
 
+### Code of Conduct
+
+On <https://github.com/parthatom/Grid/CODE_OF_CONDUCT.md> you can find the code of conduct for an efficient working of this community.
```

Comment:
You can link files in markdown by doing opening and closing brackets then opening and closing parentheses: [link example ]( ) (`[<link name>](<link url>)`